src/plugins/dam/components/batch-edit-modal/BatchEdit.jsx

```diff
@@ -0,0 +1,193 @@
+// TODO: reload the gallery-media component after the batch edit process is complete
```

Comment:
@rsumilang you can do this pretty easily by keying it to the command's id. will have to re-bind to delegate on update